Formula/mp4dovi.rb

```diff
@@ -0,0 +1,19 @@
+class Mp4dovi < Formula
+  desc "Change Dolby Vision codec in MP4 files from dvhe to dvh1"
+  homepage "https://github.com/rixtox/mp4dovi"
+  url "https://github.com/rixtox/mp4dovi/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "969703be54566fbaa586979ed2e80c68ea1db7f5e0870ac039ec1f33087b6d8d"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "-o", "mp4dovi", "mp4dovi.go"
+    bin.install "mp4dovi"
+  end
+
+  test do
+    # no test for now as a mp4 encoded in dvhe must be supplied
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
ty for the suggestion, there are some tools that I can use validating the codec after run like ffmepg or exiftool. I am thinking about write/define a simple test without the use of other media info tools.

Best